xpasses/XS002/testdata/src/a/outside_package.go

```diff
@@ -0,0 +1,24 @@
+package a
+
+import (
+	"a/schema"
+)
+
+func foutside() {
+	_ = schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {}, // want "schema attribute name should be in alphabetical order"
```

Comment:
This testing currently fails and the failure is valid 😉 

```
--- FAIL: TestXS002 (3.96s)
    analysistest.go:397: a/outside_package.go:10: no diagnostic was reported matching "schema attribute name should be in alphabetical order"
```

This file is testing a case where there may be an oddly similar package implementation of the Terraform Plugin SDK helper/schema package, but not actually that package. The comment expecting the failure can be removed. 👍 

```suggestion
			"name": {},
```